### PR TITLE
Update to section 6.5. Adding Custom RPM Repositories

### PR DESCRIPTION
**Pull request by suyogsainkar**
-label fix
-new steps

Need to add steps for custom repo creating to consider/explain restrict to Architecture & OS version feature
https://bugzilla.redhat.com/show_bug.cgi?id=1933782

Cherry-pick into:

* [x] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

**Comment by suyogsainkar** [comment]
Awaiting info from dev.

**Comment by suyogsainkar** [comment]
Applied SME review suggestions in the latest commit.

**Review by lzap — APPROVED** [review]
Thank you!

**Comment by lzap** [comment]
Oh I see second commit named WIP, is this still WIP? I suggest you use Pull Request Draft feature for WIP. If this is done, let me know and I can squash it.

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc [hunk 1]
```diff
@@ -102,11 +102,14 @@ endif::[]
 
 .Procedure
 
-. In the {Project} web UI, navigate to *Content* > *Products* and select the product that you want to use, and then click *Create Repository*.
+. In the {Project} web UI, navigate to *Content* > *Products* and select the product that you want to use, and then click *New Repository*.
 . In the *Name* field, enter a name for the repository.
 {ProjectNameX} automatically completes the *Label* field based on what you have entered for *Name*.
 . From the *Type* list, select the type of repository.
 You can select either a repository for RPM files (`yum`), Docker images (`docker`), Files (`file`) or other.
+. If `yum` content type is selected, you can restrict whether the repository is available to a host based on the host's architecture and OS version.
```
Comment (melcorr) [inline guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:110]: This isn't a step. Don't use substeps. Just integrate the non-step and the substep into one step.

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc [hunk 2]
```diff
@@ -102,11 +102,14 @@ endif::[]
...
+.. Optional: From the *Restrict to Architecture* list, select the architecture. The repository will be enabled on content hosts with the selected architecture. Select *No restriction*, which is the default value, to make the repository available to all hosts regardless of the architecture.
```
Comment (melcorr) [inline guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:111]:
```suggestion
. Optional: From the *Restrict to Architecture* list, select the architecture. Select *No restriction*, which is the default value, to make the repository available to all hosts regardless of the architecture.
```
Don't describe behaviour in steps, just the step. You can weave this into the short description if it is necessary.

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc [hunk 3]
```diff
@@ -102,11 +102,14 @@ endif::[]
...
+.. Optional: From the *Restrict to OS Version* list, select the OS version. The repository will be enabled on content hosts with the selected OS version. Select *No restriction*, which is the default value, to make the repository available to all hosts regardless of the OS version.
```
Comment (melcorr) [inline guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:112]:
```suggestion
. Optional: From the *Restrict to OS Version* list, select the OS version. Select *No restriction*, which is the default value, to make the repository available to all hosts regardless of the OS version.
```

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc (hunk 1 above)
Comment (melcorr) [inline guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:110]:
```suggestion
. If the `yum` content type is selected, you can restrict whether the repository is available to a host based on the host's architecture and OS version.
```

**Review by melcorr — CHANGES_REQUESTED** [review]
Few small requests, @suyogsainkar

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc (hunk 2 above)
Comment (jeremylenz) [reply guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:111]: If you do use this text, I'd caution against using the word "enabled." A repo can be restricted to a certain OS but still show up as "enabled" in Repository Sets on the host details screen (even though it won't be available on the host).  We should probably fix this behavior..

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc (hunk 3 above)
Comment (melcorr) [reply guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:112]: Now I read this back @suyogsainkar shouldn't it say something like 
```suggestion
.. Optional: From the *Restrict to OS Version* list, select the OS version. The repository will be enabled on content hosts with the selected OS version. Ensure that *No restriction*, which is the default value, is selected to make the repository available to all hosts regardless of the OS version.
```

If it's the default, you do nothing, just check?

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc (hunk 1 above)
Comment (suyogsainkar) [reply guides/doc-Content_Management_Guide/topics/Importing_Content.adoc]: Ack. I've integrated this text into the step above as it's related to Type. And I've changed the substeps to individual/sequential steps.

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc (hunk 2 above)
Comment (suyogsainkar) [reply guides/doc-Content_Management_Guide/topics/Importing_Content.adoc]: @melcorr @jeremylenz I've removed the text. Btw, there's also UI text which has the world "enabled".

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc (hunk 3 above)
Comment (suyogsainkar) [reply guides/doc-Content_Management_Guide/topics/Importing_Content.adoc]: Updated, thanks.

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc (hunk 1 above)
Comment (suyogsainkar) [reply guides/doc-Content_Management_Guide/topics/Importing_Content.adoc]: Fixed

**Comment by suyogsainkar** [comment]
@lzap It's not WIP anymore, I edited the PR title and removed the WIP status after my second commit. I'll consider using the PR draft feature if required going forward. thanks.

**Comment by suyogsainkar** [comment]
thanks @melcorr. I've applied your suggestions.

**Review by melcorr — APPROVED** [review]

**Comment by melcorr** [comment]
Cherry-picked to 2.3 0d918b6c0e727c446f2e79f5835104a5afc874d3

**Pull request merged**